api/services/policy/src/engine/policies/Idfm.ts

```diff
@@ -1,34 +1,46 @@
-import { NotEligibleTargetException } from '../exceptions/NotEligibleTargetException';
 import {
   OperatorsEnum,
   PolicyHandlerInterface,
   PolicyHandlerParamsInterface,
   PolicyHandlerStaticInterface,
   StatelessContextInterface,
 } from '../../interfaces';
+import { NotEligibleTargetException } from '../exceptions/NotEligibleTargetException';
 import {
   atDate,
+  endsAt,
   isAfter,
   isOperatorClassOrThrow,
   isOperatorOrThrow,
   onDistanceRange,
   onDistanceRangeOrThrow,
   perKm,
   perSeat,
-  endsAt,
   startsAt,
   watchForGlobalMaxAmount,
   watchForPersonMaxAmountByMonth,
   watchForPersonMaxTripByDay,
 } from '../helpers';
-import { ConfiguredLimitInterface, LimitTargetEnum } from '../helpers/limits';
-import { description } from './Idfm.html';
+import { LimitTargetEnum } from '../helpers/limits';
 import { AbstractPolicyHandler } from './AbstractPolicyHandler';
+import { description } from './Idfm.html';
 
 // Politique d'Île-de-France Mobilité
 /* eslint-disable-next-line */
 export const Idfm: PolicyHandlerStaticInterface = class extends AbstractPolicyHandler implements PolicyHandlerInterface {
   static readonly id = '459';
+  policy_max_amount: number;
+
+  constructor(policy_max_amount: number) {
+    super();
+    this.policy_max_amount = policy_max_amount;
```

Comment:
```suggestion
  constructor(private policy_max_amount: number) {
    super();
```

On pourrait simplifier la syntaxe comme ci-dessus. Vu que c'est du code qui est dupliqué pour chaque campagne, plus on le réduit, mieux c'est.

Reply:
J'aurais aussi tendance à ne pas utiliser le constructeur dans la classe enfant et passer les valeurs directement en propriété ou via une méthode `init`.

Reply:
Yes je fais ça !